whois-client/src/main/java/net/ripe/db/whois/common/sso/domain/HistoricalUserResponse.java

```diff
@@ -68,7 +68,7 @@ public static class AttributeChanges implements Serializable {
 
     public static class DateTimeAdapter extends XmlAdapter<String, LocalDateTime> {
 
-        private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+        private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
I double checked.
In RDAP we are using net.ripe.db.whois.api.rdap.domain.DateTimeAdapter

Same Name for the adapter but different place. I got confused because this is public and the name is the same.

Shall we make it private?? no more changes are required I think

Reply:
I'm wary of making it private because Jackon needs to call it when deserialising the response. Also then it can't be re-used. I instead renamed both classes to reflect what they do, and made them both standalone classes so they can be reused.